index.html

```diff
@@ -17,7 +17,13 @@ <h1 id="form-header">
           <label for="anrede-container">Anrede*</label>
           <ul id="anrede-container">
```

Comment:
In this case I wouldn't go with lists and list items. Most people won't, I guess.